### DISCUSSION TITLE
ETL (Embedded Template Library) and F'

### DISCUSSION THREAD

**Discussion by Gonta01** [nasa/fprime#2847] [Q&A] [answered]
Are [ETL](https://www.etlcpp.com/) and F' compatible? Does anybody have experience with using both in one project?

**Comment by LeStarch** [comment, marked as answer]
I don't see why you couldn't use ETL in a project, but I have not used it myself.  Just make sure you have proper serialization before sending ETL types through a port.

**Reply by Gonta01** [reply]
Thanks!

**Comment by Gonta01** [comment]
If anybody has some experience or faced with some difficulties while using etl with F' please share your thoughts.
